doc/examples/features_detection/plot_corner.py

```diff
@@ -17,21 +17,23 @@
 from skimage.transform import warp, AffineTransform
 from skimage.draw import ellipse
 
-
+# Sheared checkerboard
 tform = AffineTransform(scale=(1.3, 1.1), rotation=1, shear=0.7,
-                        translation=(210, 50))
-image = warp(data.checkerboard(), tform.inverse, output_shape=(350, 350))
-rr, cc = ellipse(310, 175, 10, 100)
+                        translation=(110, 30))
+image = warp(data.checkerboard()[:99, :99], tform.inverse, output_shape=(200, 310))
+# Ellipse
+rr, cc = ellipse(160, 175, 10, 100)
 image[rr, cc] = 1
-image[180:230, 10:60] = 1
-image[230:280, 60:110] = 1
+# Two squares
+image[30:80, 200:250] = 1
+image[80:130, 250:300] = 1
 
 coords = corner_peaks(corner_harris(image), min_distance=5)
 coords_subpix = corner_subpix(image, coords, window_size=13)
 
-fig, ax = plt.subplots()
-ax.imshow(image, interpolation='nearest', cmap=plt.cm.gray)
-ax.plot(coords[:, 1], coords[:, 0], '.b', markersize=3)
-ax.plot(coords_subpix[:, 1], coords_subpix[:, 0], '+r', markersize=15)
-ax.axis((0, 350, 350, 0))
+fig= plt.figure(figsize=(7, 7))
```

Comment:
This change turns object oriented plotting code into stateful code. Let's not do that.

Reply:
I rolled back to plt.subplots